lib/mpl_toolkits/axes_grid1/tests/test_axes_grid1.py

```diff
@@ -707,3 +707,36 @@ def test_removal():
     fig.canvas.draw()
     col.remove()
     fig.canvas.draw()
+
+
+@image_comparison(['anchored_locator_base_call.png'], style="mpl20")
+def test_anchored_locator_base_call():
+    fig = plt.figure(figsize=(3, 3))
+    fig1, fig2 = fig.subfigures(
+        nrows=2, ncols=1, height_ratios=[1, 1], squeeze=True
+    )
+
+    ax = fig1.subplots()
+    ax.set(aspect=1, xlim=(-15, 15), ylim=(-20, 5))
+
+    Z = cbook.get_sample_data(
+        "axes_grid/bivariate_normal.npy", np_load=True
+    )
+    extent = (-3, 4, -4, 3)
+
+    axins = zoomed_inset_axes(ax, zoom=2, loc="upper left")
+    axins.set(xticks=[], yticks=[])
+    im = axins.imshow(Z, extent=extent, origin="lower")
+
+    # colorbar
+    cax = inset_axes(
+        axins,
+        width="5%",  # width = 10% of parent_bbox width
+        height="100%",  # height : 50%
```

Comment:
```suggestion
        width="5%",
        height="100%",
```
or am I missing the point of the comments? It seems like they are not correct?